src/main/java/org/cascadebot/cascadebot/UnicodeConstants.java

```diff
@@ -48,6 +48,33 @@
     public static final String EIGHT = "\u0038\u20E3"; // 8️⃣
     public static final String NINE = "\u0039\u20E3"; // 9️⃣
 
+    public static final String A = "\uD83C\uDDE6";
+    public static final String B = "\uD83C\uDDE7";
+    public static final String C = "\uD83C\uDDE8";
+    public static final String D = "\uD83C\uDDE9";
+    public static final String E = "\uD83C\uDDEA";
+    public static final String F = "\uD83C\uDDEB";
+    public static final String G = "\uD83C\uDDEC";
+    public static final String H = "\uD83C\uDDED";
+    public static final String I = "\uD83C\uDDEE";
+    public static final String J = "\uD83C\uDDEF";
+    public static final String K = "\uD83C\uDDF0";
+    public static final String L = "\uD83C\uDDF1";
+    public static final String M = "\uD83C\uDDF2";
+    public static final String N = "\uD83C\uDDF3";
+    public static final String O = "\uD83C\uDDF4";
+    public static final String P = "\uD83C\uDDF5";
+    public static final String Q = "\uD83C\uDDF6";
+    public static final String R = "\uD83C\uDDF7";
+    public static final String S = "\uD83C\uDDF8";
+    public static final String T = "\uD83C\uDDF9";
+    public static final String U = "\uD83C\uDDFA";
+    public static final String V = "\uD83C\uDDFB";
+    public static final String W = "\uD83C\uDDFC";
+    public static final String X = "\uD83C\uDDFD";
+    public static final String Y = "\uD83C\uDDFE";
+    public static final String Z = "\uD83C\uDDFF";
```

Comment:
Following the standard in UnicodeConstants, we should probably have a comment with the emoji